typings/index.d.ts

```diff
@@ -1357,11 +1357,11 @@ export class Message<Cached extends boolean = boolean> extends Base {
   public embeds: MessageEmbed[];
   public groupActivityApplication: ClientApplication | null;
   public guildId: Snowflake | null;
-  public readonly guild: Guild | null;
+  public readonly guild: If<Cached, Guild, null>;
```

Comment:
```suggestion
  public readonly guild: If<Cached, Guild>;
```
Since the default value is `null`, the parameter is not needed. This is also consistent with other uses of that type.